.storybook/themes/commerce.scss

```diff
@@ -390,10 +445,36 @@
 				quilted-image__wrapper: (
 					grid-template-areas: "top top" "bottom bottom",
 				),
+				story-carousel: (
+					components: (
+						carousel: (
+							--slides: 4,
+						),
+						carousel-actions: (
+							display: flex,
+							margin-top: -7rem,
+						),
+						carousel-track: (
+							justify-self: center,
+							max-width: 88%,
+						),
+					),
+				),
+				category-carousel-title: (
+					components: (
+						heading: (
+							font-size: var(--global-font-size-7),
+							line-height: var(--global-line-height-4),
+							font-weight: var(--global-font-weight-7),
+						),
+					),
+					padding-left: 6rem,
```

Comment:
If you set the width of the header on default to `var(--layout-content-scale-width)` with auto margins it should match the track given the new defaults.